crates/aptos/src/common/types.rs

```diff
@@ -823,7 +831,10 @@ impl RestOptions {
     }
 
     pub fn client(&self, profile: &str) -> CliTypedResult<Client> {
-        Ok(Client::new(self.url(profile)?))
```

Comment:
here, implementation is:


    pub fn new(base_url: Url) -> Self {
        Self::new_with_timeout(base_url, Duration::from_secs(10))
    }